packages/core/src/lib/tasks/task.service.ts

```
@@ -743,66 +743,56 @@ export class TaskService extends TenantAwareCrudService<Task> {
-			// Find all assigned tasks of employee
-			const tasks = await query.getMany();
+			// Find all assigned tasks of employee with relations
+			const tasks = await query
+				.leftJoinAndSelect(`${query.alias}.members`, 'task_members')
+				.leftJoinAndSelect(`${query.alias}.teams`, 'task_teams')
+				.getMany();
 
 			// Unassign member from All the Team Tasks
 			tasks.forEach((task) => {
-				if (task.teams.length) {
+				if (task.teams && task.teams.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)